homework-submission/week3_app/src/App.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import './App.css';
+import Steps from './components/FetchOrigin';
```

Comment:
Better to name your file like the component it contains, this is a bit confusing

Reply:
yes... thanks for the notification.